cpp/src/example/example_cluster_mode.cc

```diff
@@ -0,0 +1,46 @@
+
+/// This is a complete example of writing a distributed program using the C ++ worker API.
+
+/// including the header
+#include <ray/api.h>
+#include <ray/api/ray_config.h>
+#include <ray/util/logging.h>
+
+/// using namespace
+using namespace ray::api;
+
+/// general function of user code
+int Return1() { return 1; }
+int Plus1(int x) { return x + 1; }
+int Plus(int x, int y) { return x + y; }
+
+int main(int argc, char **argv) {
+  RAY_LOG(INFO) << "Start cpp worker example";
+
+  /// initialization to cluster mode
+  ray::api::RayConfig::GetInstance()->run_mode = RunMode::CLUSTER;
+  /// Set redis ip to connect an existing ray cluster.
+  ray::api::RayConfig::GetInstance()->redis_ip = "127.0.0.1";
+  ray::api::RayConfig::GetInstance()->lib_name =
+      "/Users/jiulong/arcos/opensource/ant/ray/bazel-bin/cpp/example_cluster_mode.so";
+  Ray::Init();
+
+  /// put and get object
+  auto obj = Ray::Put(123);
+  auto get_result = *(obj.Get());
```

Comment:
Can we use here your new feature for Ray::Get? (Perhaps mix Ray::Get and obj.get() or add `// Ray::Get(..)` works too.)

Reply:
Sure.